packages/waku/src/router/define-router.ts

```diff
@@ -269,3 +269,33 @@ export function unstable_rerenderRoute(
   const rscPath = getRscPath(pathname);
   rerender(rscPath, { query, skip });
 }
+
+// -----------------------------------------------------
+// new_defineRouter
+// Eventually replaces unstable_defineRouter
+// -----------------------------------------------------
+
+export function new_defineRouter(
+  getPathConfig: () => Promise<
+    Iterable<{
+      pattern: string; // TODO let's revisit this later
+      path: PathSpec;
+      components: Record<
+        string, // componentId such as "**/layout" or "**/page" in the case of createPages
+        {
+          isStatic?: boolean;
+        }
+      >;
+      noSsr?: boolean;
+    }>
+  >,
+  renderRoute: (
+    path: string, // "/foo/bar"
+    options: {
+      query?: string;
+      skipIds?: string[];
```

Comment:
These are component ids specific to RSC level components, correct? 

Sidenote: I do see not that the server does need to be "told" somehow about which RSCs already exist on the client router, which helps explain the use of `skipIds`

Reply:
It's the subset of strings specified at L284.